package.json

```diff
@@ -24,7 +24,7 @@
     "public-api:update": "api-extractor run -l",
     "start": "ng serve clr-demo",
     "test": "ng test clr-angular --source-map=false --watch=false",
-    "test:watch": "npm run --silent _test:angular -- --watch",
+    "test:watch": "ng test clr-angular --watch",
```

Comment:
This is fixed in https://github.com/vmware-clarity/ng-clarity/pull/23, so please remove this from this PR.

Reply:
Done